packages/editor/src/lib/hooks/usePattern.tsx

```diff
@@ -88,7 +88,7 @@ const getDefaultPatterns = () => {
 export const usePattern = () => {
 	const app = useApp()
 	const dpr = app.devicePixelRatio
-	const [isReady, setIsReady] = useState(false)
+	const [isReady, setReady] = useState(false)
```

Comment:
The norm is `something` / `setSomething` why the change?

Reply:
Fixed!